services/src/tasks/mod.rs

```diff
@@ -257,21 +258,31 @@ impl TaskStatus {
             _ => String::new(),
         }
     }
+
+    fn time_started(&self) -> TimeInstance {
+        match self {
+            TaskStatus::Running(info) => info.time_estimate.time_started(),
+            _ => TimeInstance::MIN,
```

Comment:
can you return an Option here instead? And user `expect` or `ok_or()` on it where you need the value?

Reply:
[ad1fcb8](https://github.com/geo-engine/geoengine/pull/706/commits/ad1fcb824d1237e4de62a7080702481a6294607e)